src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -42,58 +55,104 @@
  * execution times significantly if applicable.
  * <p>
  */
-public class CommonTermsQueryBuilder extends QueryBuilder implements BoostableQueryBuilder<CommonTermsQueryBuilder> {
+public class CommonTermsQueryBuilder extends QueryBuilder<CommonTermsQueryBuilder> implements BoostableQueryBuilder<CommonTermsQueryBuilder> {
 
     public static final String NAME = "common";
 
+    public static final float DEFAULT_CUTOFF_FREQ = 0.01f;
+
+    public static final Operator DEFAULT_HIGH_FREQ_OCCUR = Operator.OR;
+
+    public static final Operator DEFAULT_LOW_FREQ_OCCUR = Operator.OR;
+
+    public static final boolean DEFAULT_DISABLE_COORD = true;
+
     public static enum Operator {
```

Comment:
@MaineC can you have a look too? this could be shared between different queries right? should we extract it from here and re-use in simple query string for instance? can you review the enum and what it looks like?